setup.cfg

```diff
@@ -0,0 +1,8 @@
+[pycodestyle]
```

Comment:
I think we shouldn't add this file. People can have their own configurations for whatever code quality tools they use.